src/components/v5/common/MemberSignatureList/types.ts

```diff
@@ -0,0 +1,11 @@
+import { MemberSignatureProps } from '../MemberSignature/types';
+
+export interface MemberSignatureListProps {
+  items: MemberSignatureItem[];
+  isLoading?: boolean;
+  title: React.ReactNode;
+}
+
+export interface MemberSignatureItem extends MemberSignatureProps {
```

Comment:
You're using this interface in the interface above. The order should be different. Please move this one to the top of this file, below the imports